File: week-03/day3/6.py
# create a function that takes a list and returns a new list that is reversed

easy_list = [1, 2, 3, 4, 5, 6, 7, 8, 9, 0]

def list_reverser(input):
    reversed_list = input[::-1]
    return reversed_list

print(list_reverser(easy_list))

File: week-07/day2/6.js
'use strict';

// create a function that takes a string and a letter and returns a boolean
// it should return true if the string consits the given letter, false otherwise

var string = 'qwertzuiopasdfghjklyxcvbnm'


function is_element_in_array(string, letter){
    return string.indexOf(letter) !== -1;
}

console.log(is_element_in_array(string, 'a'));
console.log(is_element_in_array(string, 'é'));
